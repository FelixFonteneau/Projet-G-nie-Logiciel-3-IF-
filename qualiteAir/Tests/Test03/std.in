3
1
01/01/2017 00:00
01/01/2017 01:00
3
2
-8
80
15000
01/01/2017 00:00
01/01/2017 01:00
3
1
01/01/2017 00:00
01/01/2017 00:01
7
