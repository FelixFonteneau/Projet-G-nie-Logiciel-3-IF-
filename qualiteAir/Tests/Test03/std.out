Bonjour, et bienvenue dans cette application de surveillance de la qualité de l'air.
Le service est en cours d'initialisation, veuillez patientez...

Nombre de mesures analysées : 41

Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


1 - Effectuer la requête sur tous les capteurs
2 - Effectuer la requête sur les capteurs d'un territoire délimité


Quelle est la date de début (Au format JJ/MM/AAAA HH:MM) ?


Quelle est la date de fin (Au format JJ/MM/AAAA HH:MM) ?


           | Capteur 0 | Capteur 1 | Capteur 2 | Capteur 3 | Capteur 4 |
------------------------------------------------------------------------
Capteur 0      100%         50%         54%         36%        100%
------------------------------------------------------------------------
Capteur 1                  100%         40%          0%         49%
------------------------------------------------------------------------
Capteur 2                              100%         49%         70%
------------------------------------------------------------------------
Capteur 3                                          100%         48%
------------------------------------------------------------------------
Capteur 4                                                      100%
------------------------------------------------------------------------

(Un score de -1% signifie qu'aucune donnée n'a été trouvée sur cette periode là)

Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


1 - Effectuer la requête sur tous les capteurs
2 - Effectuer la requête sur les capteurs d'un territoire délimité


Quelle est la latitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -90 et 90.


Quelle est la longitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -180 et 180.


Quel est le rayon (en km) ? Format : 'nombres' et '.' seulement acceptés et radius > 0.


Quelle est la date de début (Au format JJ/MM/AAAA HH:MM) ?


Quelle est la date de fin (Au format JJ/MM/AAAA HH:MM) ?


           | Capteur 0 | Capteur 1 | Capteur 2 |
------------------------------------------------
Capteur 0      100%          0%        100%
------------------------------------------------
Capteur 1                  100%         19%
------------------------------------------------
Capteur 2                              100%
------------------------------------------------

(Un score de -1% signifie qu'aucune donnée n'a été trouvée sur cette periode là)

Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


1 - Effectuer la requête sur tous les capteurs
2 - Effectuer la requête sur les capteurs d'un territoire délimité


Quelle est la date de début (Au format JJ/MM/AAAA HH:MM) ?


Quelle est la date de fin (Au format JJ/MM/AAAA HH:MM) ?


           | Capteur 0 | Capteur 1 | Capteur 2 | Capteur 3 | Capteur 4 |
------------------------------------------------------------------------
Capteur 0      100%         -1%         -1%         -1%         -1%
------------------------------------------------------------------------
Capteur 1                  100%         75%          0%         99%
------------------------------------------------------------------------
Capteur 2                              100%         37%        100%
------------------------------------------------------------------------
Capteur 3                                          100%         56%
------------------------------------------------------------------------
Capteur 4                                                      100%
------------------------------------------------------------------------

(Un score de -1% signifie qu'aucune donnée n'a été trouvée sur cette periode là)

Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


Merci et à bientôt !
